src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/RetrieveBurdenEstimatesTests.kt

```diff
@@ -373,6 +375,63 @@ class RetrieveBurdenEstimatesTests : BurdenEstimateRepositoryTests()
         }
     }
 
+    @Test
+    fun `can getBurdenEstimateOutcomesSequence`()
+    {
+        val setId = withDatabase { db ->
+            val ids = setupDatabase(db)
+            val modelVersionId = ids.modelVersion!!
+            db.addBurdenEstimateSet(ids.responsibility, modelVersionId, username)
+        }
+        withDatabase{ db ->
+            db.addCountries(listOf("ABC", "DEF"))
+            db.addBurdenEstimate(setId, "DEF", 2001, 21, "cohort_size", 5f )
+            db.addBurdenEstimate(setId, "ABC", 2000, 20, "deaths", 10f )
+        }
+        withRepo { repo ->
+            val result = repo.getBurdenEstimateOutcomesSequence(groupId,
+                    touchstoneVersionId, scenarioId, setId).toList()
+
+            Assertions.assertThat(result.count()).isEqualTo(2);
+
+            Assertions.assertThat(result[0].disease).isEqualTo("Hib3")
+            Assertions.assertThat(result[0].year).isEqualTo(2000)
+            Assertions.assertThat(result[0].age).isEqualTo(20)
+            Assertions.assertThat(result[0].country).isEqualTo("ABC")
+            Assertions.assertThat(result[0].countryName).isEqualTo("ABC-Name")
+            Assertions.assertThat(result[0].burden_outcome_code).isEqualTo("deaths")
+            Assertions.assertThat(result[0].value).isEqualTo(10f)
+
+            Assertions.assertThat(result[1].disease).isEqualTo("Hib3")
+            Assertions.assertThat(result[1].year).isEqualTo(2001)
+            Assertions.assertThat(result[1].age).isEqualTo(21)
+            Assertions.assertThat(result[1].country).isEqualTo("DEF")
+            Assertions.assertThat(result[1].countryName).isEqualTo("DEF-Name")
+            Assertions.assertThat(result[1].burden_outcome_code).isEqualTo("cohort_size")
+            Assertions.assertThat(result[1].value).isEqualTo(5f)
+
+        }
+    }
+
+    @Test
+    fun `can getExpectedOutcomesForBurdenEstimateSet`()
+    {
```

Comment:
similarly think we need a test here that only data for the desire set is returned